bears/php/PHPMessDetectorBear.py

```diff
@@ -0,0 +1,43 @@
+from coalib.settings.Setting import typed_list
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import \
+    DistributionRequirement
+
+
+@linter(executable='phpmd',
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>\d+)(?P<message>.*)')
+class PHPMessDetectorBear:
+    """
+    The bear takes a given PHP source code base and looks for several
+    potential problems within that source. These problems can be things like:
+
+    - Possible bugs
+    - Suboptimal code
+    - Overcomplicated expressions
+    - Unused parameters, methods, properties
+    """
+
+    LANGUAGES = {'PHP'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='phpmd')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Complexity', 'Unused Code', 'Redundancy',
+                  'Variable Misuse'}
+    SEE_MORE = 'https://phpmd.org/about.html'
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         phpmd_ruleset: typed_list(str)=''):
```

Comment:
this is wrong on many levels.

never use an empty string as a default unless that is a valid value.  Use `None`.

and this is supposed to be a list, but you give a string as a default.

But, since it is a mandatory argument, simply do not provide a default and coala will do the work for you.